src/main/java/model/Book.java

```diff
@@ -50,7 +50,7 @@ public class Book {
     private String description;
     @Column(name = "cover_image")
     private String coverImage;
-    @Column(nullable = false)
+    @Column(name = "is_deleted", nullable = false)
     private boolean isDeleted = false;
```

Comment:
In my project Hibernate create it by default, without explicit indication

![image](https://github.com/Alexander1iubchenko/Book-store/assets/94116191/18d3a495-ba8e-4e98-a82f-b535d4d22b21)